lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -58,6 +58,14 @@ void ConfigBuilder::buildPalMetadata() {
     if (!m_pipelineState->isWholePipeline() && m_pipelineState->hasShaderStage(ShaderStageFragment)) {
       // FS-only shader or part-pipeline compilation
       buildPipelineVsFsRegConfig();
+    } else if (m_hasTask || m_hasMesh) {
+      if (m_hasTask) {
+        // Task-Mesh-FS pipeline
+        buildPipelineTaskMeshFsConfig();
+      } else {
+        // Mesh-FS pipeline
+        buildPipelineMeshFsConfig();
+      }
```

Comment:
nit: I think it'd be more straightforward to have these two as two separate top-level `else if` statements